linux/kernel/building.md

```diff
@@ -125,6 +125,11 @@ If you are using a 32-bit operating system (for example, our Raspberry Pi Deskto
 
 `sudo apt install zlib1g-dev:amd64`
 
+If you are using CCache and a CI environment, instruct it to not use compiler's mtime for cache ID calculations.
+This is because Git intentionally doesn't save file timestamps, so each time you clone the toolchain, its file mtimes are different, invalidating CCache's cache with default settings.
```

Comment:
Perhaps it's just me, but "invalidating CCache's cache with default settings" makes it sounds like it's the "default settings" that are going into the cache, and making it invalid? Perhaps a small tweak to:
"invalidating CCache's cache when default settings are used" ? (and perhaps the comma after "toolchain" can be removed?)

Reply:
Fine by me. Overkill or not, it's only better if the formulations are clearer.